server/api/static.js

```diff
@@ -0,0 +1,41 @@
+import { readFile } from 'fs'
+import path from 'path'
+import { URL } from 'node:url'
+
+const CLIENT_DIR = '../../client'
+const DIST_DIR = '/dist'
+const ASSETS_DIR = '/assets'
```

Comment:
We will not use `assets` because of performance issues (we use it for now, but we already have an issue to replace it).

Let’s avoid it in this code.

Reply:
Hm, but we need to support `favicon.ico` without cache busters. So maybe we will need `assets`.